internal/actor/organisation.go

```diff
@@ -28,13 +28,26 @@ type Member struct {
 	CreatedAt time.Time
 	// UpdatedAt is when the Member was last updated
 	UpdatedAt time.Time
+	// LastLoggedInAt is when the Member last logged in to the service
+	LastLoggedInAt time.Time
+	Email          string
+	FirstNames     string
+	LastName       string
+	// Permission is the type of permissions assigned to the member to set available actions in an Organisation
+	Permission Permission
+}
+
+func (i Member) FullName() string {
+	return fmt.Sprintf("%s %s", i.FirstNames, i.LastName)
 }
 
 // A MemberInvite is created to allow a new Member to join an Organisation
 type MemberInvite struct {
 	PK, SK string
 	// CreatedAt is when the MemberInvite was created
 	CreatedAt time.Time
+	// UpdatedAt is when the MemberInvite was last updated
+	UpdatedAt time.Time
```

Comment:
One downside of relying on`SkUpdatedAtIndex` but not the end of the world.

Reply:
it's fine, we should be consistent with these, even if we aren't changing an entity